pinot-controller/src/main/resources/package-lock.json

```diff
@@ -12,14 +12,6 @@
         "@babel/highlight": "^7.16.7"
       }
     },
-    "@babel/helper-module-imports": {
```

Comment:
revert this file?

Reply:
urgh.. i thought I removed this from the commit. will fix it. tks!